### PR TITLE
docs: refresh getting started docs

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/why-playwright.md [hunk 1]
```diff
@@ -35,27 +37,8 @@ Playwright enables fast, reliable and capable automation across all modern brows
...
-  ```
-
-* **TypeScript support**. Playwright ships with built-in types for auto-completion and other benefits.
-
-* **Debugging tools**. Playwright works with the [editor debugger and browser developer tools](./debug.md) to pause execution and inspect the web page.
-
-* **Language bindings**. Playwright is available for [Node.js](https://github.com/microsoft/playwright) [Python](https://github.com/microsoft/playwright-python), [.NET](https://github.com/microsoft/playwright-dotnet) and
-[Java](https://github.com/microsoft/playwright-java). Learn more about [supported languages](./languages.md).
-
-* **Deploy tests to CI**. First-party [Docker image](./docker.md) and [GitHub Actions](./ci.md#github-actions) support to deploy tests to [your preferred CI/CD provider](./ci.md).
```
Comment (mxschmitt) [inline docs/src/why-playwright.md:55]: realisation: I see over and over people who didn't know that we have a Docker image.

File: docs/src/ci.md [hunk 2]
```diff
@@ -312,7 +312,7 @@ tests:
 
 By default, Playwright downloads browser binaries when the Playwright NPM package
 is installed. The NPM packages have a `postinstall` hook that downloads the browser
-binaries. This behavior can be [customized with environment variables][installation-managing-browser-binaries].
```
Comment (mxschmitt) [inline docs/src/ci.md:315]: please follow up on playwright.dev and remove this weird link which I've created in the params when rolling.

File: docs/src/installation.md [hunk 3]
```diff
@@ -1,316 +0,0 @@
----
-id: installation
-title: "Installation"
----
-
-<!-- TOC -->
-
-### Prerequisites for .NET
```
Comment (mxschmitt) [inline docs/src/installation.md:8]: If I see this correctly, this part was fully nuked out of the docs. So there is no mentioning anywhere anymore about the Playwright .NET CLI tool thingy.

File: docs/src/why-playwright.md (hunk 1 above)
Comment (pavelfeldman) [reply docs/src/why-playwright.md:55]: I don't think this page would help them though...

File: docs/src/ci.md (hunk 2 above)
Comment (pavelfeldman) [reply docs/src/ci.md:315]: Yes, I started there.

File: docs/src/installation.md (hunk 3 above)
Comment (pavelfeldman) [reply docs/src/installation.md:8]: Yeah, let me restore it...

**Pull request merged**